cohere/client.py

```diff
@@ -334,23 +318,20 @@ def summarize(
             additional_command (str): (Optional) Modifier for the underlying prompt, must \
```

Comment:
Might need to update the docstring to be "Returns a generated summary" instead of "Return ..." to be consistent with the other docs